.github/workflows/build-and-test-macos.yml

```diff
@@ -53,6 +53,10 @@ jobs:
         run: |
           brew install pkg-config
 
+      - uses: julia-actions/setup-julia@v2
+        with:
+          version: '1'
```

Comment:
Is there a recommendation from julia whether to also pin subversion (1.10 / 1.11)? I know these can have some breaking changes, although our usage might be light enough not to worry.

Reply:
I think I'd rather let it run along at 1 until there's a bug to prevent us otherwise. That way we'll naturally incorporate any updates. Our usage is very light anyway, so should be fine I think.